lib/samples/show_legend/show_legend_sample.dart

```diff
@@ -0,0 +1,174 @@
+//
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+import '../../utils/sample_state_support.dart';
+
+class ShowLegendSample extends StatefulWidget {
+  const ShowLegendSample({super.key});
+  @override
+  State<ShowLegendSample> createState() => _ShowLegendSampleState();
+}
+
+class _ShowLegendSampleState extends State<ShowLegendSample>
+    with SampleStateSupport {
+  // create a map view controller
```

Comment:
We decided to standardize comments with a capital letter and a period --
```dart
// Create a map view controller.
```
We should apply that to the comments throughout.